### PR TITLE
Fixed update issues

### PR DESCRIPTION
**Pull request by codingvlad**
I managed to fix the update issue. This is one way of doing it. One should be aware that this might not be the cleanest way of doing it. Further research is needed.

### CODE REVIEW

File: range-slider/src/app/app.component.ts [hunk 1]
```diff
@@ -11,8 +12,20 @@ export class AppComponent {
   public someMin: number = -10;
   public someMax: number = 10;
 
+  @ViewChild('slider') slider: NouisliderComponent;
+
   onChange(value: any) {
     console.log('Value changed to', value);
+    this.someValue = [value[0], this.someValue[1]];
+  }
```
Comment (JoshSimon) [inline range-slider/src/app/app.component.ts:20]: What about 
`this.someValue[0]=value[0]` ?

Is it necessary to assign both entries to someValue, even if only one changed?

File: range-slider/src/app/app.component.html [hunk 2]
```diff
@@ -1,22 +1,11 @@
...
-  <div class="right-input-div">
-    <input class="right-input" type="number" value="someValue(1)" [(ngModel)]="someValue">
-  </div>-->
   <form class="form-horizontal">
     <div class="form-group">
       <div class="col-sm-2">
         <input class="form-control" [min]="someMin" [max]="someMax" name="min" [(ngModel)]="someValue[0]" type="number" (change)="onChange0(someValue)" (input)="onChange0(someValue)">
       </div>
       <div class="col-sm-8">
-        <nouislider #slider [connect]="true" name="slider" id="slider" [min]="someMin" [max]="someMax" [config]="someKeyboardConfig" [(ngModel)]="someValue"
```
Comment (JoshSimon) [inline range-slider/src/app/app.component.html]: Why is `#slider` not necessary?

File: range-slider/src/app/app.component.html [hunk 3]
```diff
@@ -1,22 +1,11 @@
...
-      [disabled]="disabled"></nouislider>
-  </div>
-  <div class="right-input-div">
-    <input class="right-input" type="number" value="someValue(1)" [(ngModel)]="someValue">
-  </div>-->
   <form class="form-horizontal">
     <div class="form-group">
       <div class="col-sm-2">
         <input class="form-control" [min]="someMin" [max]="someMax" name="min" [(ngModel)]="someValue[0]" type="number" (change)="onChange0(someValue)" (input)="onChange0(someValue)">
       </div>
```
Comment (JoshSimon) [inline range-slider/src/app/app.component.html]: Why are `(change)` and `(input)` calling both the same function?

File: range-slider/src/app/app.component.ts (hunk 1 above)
Comment (JoshSimon) [reply range-slider/src/app/app.component.ts:20]: Does it really make sense to treat `value` as an array? I guess you could input a number through the HTML input elements and assign it to the dedicated entry in `someValue`.

Comment (codingvlad) [reply range-slider/src/app/app.component.ts:20]: You could do that absolutely (this.someValue[0]=value[0]). What I did was not change too much your code. As I said there are many ways of doing it. We should discuss for a more idiomatic solution. As for the array, I think is something that nouislider needs. If the component expects that datatype one should use that.

File: range-slider/src/app/app.component.html (hunk 2 above)
Comment (codingvlad) [reply range-slider/src/app/app.component.html]: You do not use in your component (@ViewChild remember?) #slider is another way of writing a directive.

File: range-slider/src/app/app.component.html (hunk 3 above)
Comment (codingvlad) [reply range-slider/src/app/app.component.html]: `change` gets triggered when the user enters a number and hits return and `input` is triggered when the arrows of the number input are clicked.

**Pull request merged**